### ISSUE TITLE
compile:file not working with erlang 27

### ISSUE THREAD

**Issue by skogsbaer** [erlang/otp#8650]
#### The bug

After installing erlang 27 with kerl, I am not able to parse files from the erlang distribution using `compile:file`. 

Error message:

```
/usr/local/lib/kerl-erlang/27.0/lib/erts-15.0/src/erlang.erl:9778:2: can't find doc file "../../doc/src/erlang_system_info.md"
```

The same code worked with erlang 26.

No `.md` file is found under `/usr/local/lib/kerl-erlang/27.0`.

#### Software versions

* Operating System: Mac OSX 14.5
* Erlang/OTP: 27
* Installed erlang with`kerl` version 4.0.0

We also experienced this bug under Linux, erlang was not installed with `kerl` in this case. Hence, I do not believe it's a distribution problem. @albsch can you add details?

#### How to replicate

An ordered list of steps to replicate the bug, e.g.:

1. run `KERL_BUILD_DOCS=yes kerl install 27.0-builtdocs /usr/local/lib/kerl-erlang/27.0
2. execute `erl`
3. Then

```
$ erl
Erlang/OTP 27 [erts-15.0] [source] [64-bit] [smp:11:11] [ds:11:11:10] [async-threads:1] [jit]

Eshell V15.0 (press Ctrl+G to abort, type help(). for help)
2> compile:file("/usr/local/lib/kerl-erlang/27.0/lib/erts-15.0/src/erlang", [report]).
/usr/local/lib/kerl-erlang/27.0/lib/erts-15.0/src/erlang.erl:9778:2: can't find doc file "../../doc/src/erlang_system_info.md"
% 9778| -doc {file,"../../doc/src/erlang_system_info.md"}.
%     |  ^

/usr/local/lib/kerl-erlang/27.0/lib/erts-15.0/src/erlang.erl:11582:2: can't find doc file "../../doc/src/erlang_port_info.md"
% 11582| -doc {file, "../../doc/src/erlang_port_info.md"}.
%      |  ^

error
```

#### Expected behaviour

The file is parsed successfully. Here is what happens for Erlang 26:

```
$ erl
Erlang/OTP 26 [erts-14.2.2] [source] [64-bit] [smp:11:11] [ds:11:11:10] [async-threads:1] [jit] [dtrace]

Eshell V14.2.2 (press Ctrl+G to abort, type help(). for help)
1> compile:file("/opt/homebrew/Cellar/erlang/26.2.2/lib/erlang/lib/erts-14.2.2/src/erlang", [report]).
{ok,erlang}
```

**Comment by paulo-ferraz-oliveira** [comment]
@skogsbaer, are you using any Kerl options not present in the commands above (like exported env. variables)? Or does your `~/.kerlrc` have content you can share. That'd hopefully help in debugging.

**Comment by skogsbaer** [comment]
@paulo-ferraz-oliveira No, I did not tweak kerl, I don't even have a `~/.kerlrc` file.

**Comment by albsch** [comment]
I have the same issue but with `erlang` and `erlang-docs` installed via `pacman` in a `arch-linux` distribution. I first thought it has something to do with how arch packages `erlang`, but I can't find any `.md` documentation file anywhere. So they seem to indeed be missing. My `/usr/lib/erlang/erts-15.0/doc` folder is empty, for example.

**Comment by jhogberg** [comment]
This should be fixed now that `27.0.1` is released, thanks for your report!
